Mathlib/LinearAlgebra/Prod.lean

```diff
@@ -239,6 +239,11 @@ theorem coprod_map_prod (f : M →ₗ[R] M₃) (g : M₂ →ₗ[R] M₃) (S : Su
     rw [← Set.image2_add, Set.image2_image_left, Set.image2_image_right]
     exact Set.image_prod fun m m₂ => f m + g m₂
 
+@[simp]
+theorem coprod_comp_inl_inr (f : M × M₂ →ₗ[R] M₃) :
+    (f.comp (LinearMap.inl R M M₂)).coprod (f.comp (LinearMap.inr R M M₂)) = f := by
```

Comment:
```suggestion
    (f.comp (inl R M M₂)).coprod (f.comp (inr R M M₂)) = f := by
```
untested, but it seems to work in earlier lemmas `fst_eq_coprod` and `coprod_inl_inr`